### PR TITLE
fix(chat): fix button alignment for prompt review mode, filters UI (Issues #1696, #1700, #1701)

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
**Description:**

- fix button alignment for prompt review mode
- fix filters tooltip font size and height
- fix filters input size
- fix rules comparison

Issues:

- #1696 
- #1700 
- #1701 
- #1709 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1705.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1705.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — DISMISSED** [review]

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1705.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat/src/components/Chat/Publish/HandlePublication.tsx [hunk 1]
```diff
@@ -36,6 +36,7 @@ import {
 } from './PublicationResources';
 import { RuleListItem } from './RuleListItem';
 
+import { isEqual } from 'lodash-es';
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/Publish/HandlePublication.tsx]:
```suggestion
import isEqual from 'lodash-es/isEqual';
```

**Review by IlyaBondar — DISMISSED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1705.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**